protocol-units/bridge/sources/METHBridge.move

```diff
@@ -0,0 +1,455 @@
+module 0x1::METHBridge {
```

Comment:
Could be great to get some documentation here for the module and maybe name it, something like `native_bridge::moveth`